eth/backend.go

```diff
@@ -627,6 +628,9 @@ func (s *Ethereum) Start() error {
 	s.handler.Start(maxPeers)
 
 	go s.startCheckpointWhitelistService()
+	go s.startMilestoneWhitelistService()
```

Comment:
Could you check that we'll close all 4 this goroutines if we close *Ethereum?

Reply:
Yes, they all close down using choseCh channel